app/assets/stylesheets/common/components/signup-progress-bar.scss

```diff
@@ -0,0 +1,81 @@
+$progress-bar-line-width: 2px;
+$progress-bar-circle-size: 1.2rem;
+$progress-bar-icon-size: 0.8rem;
+
+.signup-progress-bar {
+  width: 100%;
+  display: flex;
+  color: var(--primary-low-mid);
+  box-sizing: border-box;
+  margin-bottom: 1.2em;
+
+  &__segment {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+
+    &:last-child {
+      width: $progress-bar-circle-size;
+    }
+  }
+
+  &__step-text {
+    white-space: nowrap;
+    width: fit-content;
+    transform: translateX(-25%);
+
+    .signup-progress-bar__segment:first-child & {
+      transform: translateX(0%);
+    }
+
+    .signup-progress-bar__segment:last-child & {
+      transform: translateX(-50%);
+    }
+  }
+
+  &__step {
+    display: flex;
+  }
+
+  &__line {
+    transform: translateY(
+      calc(calc($progress-bar-circle-size + $progress-bar-line-width) / 2)
+    );
+    height: $progress-bar-line-width;
+    width: 100%;
+    background-color: var(--primary-low-mid);
+  }
+
+  &__circle {
```

Comment:
If we can add this to first-child (first circle)
```
transform: translateX(50%);
z-index: 1; 
```
and equivalent to the last circle, it aligns more nicely with the text
![CleanShot 2024-08-26 at 18 52 02@2x](https://github.com/user-attachments/assets/112449b3-3f78-4f08-9cb4-709f9f9975ce)

Reply:
This is a bit tricky. translateX(50%) works well for English as it seems to center the text, but that's not necessarily the case for other languages:

![image](https://github.com/user-attachments/assets/00f19cb5-ec41-4663-a784-05cdd96a7a73)

That's the reason the first and last were initially not centrally aligned, as we don't know how long the text could be. Another option would be to center the circle with the text, but that would make the steps not evenly spaced, so I think there's always a compromise. I added the transform anyway; I like the "padding" it adds, even if it's not centered